src/olympia/devhub/views.py

```diff
@@ -1397,6 +1397,17 @@ def submit_version_distribution(request, addon_id, addon):
         ),
         False,
     ),
+    (
+        'tab_line',
+        _('Tab color highlight'),
```

Comment:
Let's just call it 'Tab highlight', since we don't use 'color' for any of the others (and they're all colors).

Reply:
https://github.com/mozilla/addons-server/pull/17193/commits/1c913fb3ad44201d3190a67f66d064d7a0b755ec